CMakeLists.txt

```diff
@@ -33,13 +22,9 @@ INCLUDE_DIRECTORIES(
         /usr/include
 )
 
-if(CMAKE_BUILD_TYPE STREQUAL "")
-    set(CMAKE_BUILD_TYPE Release)
-endif()
-
-set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Wall -std=${CMAKE_CXX_STD_FLAGS}")
-set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -Wall -std=${CMAKE_CXX_STD_FLAGS}")
-
+IF(CMAKE_BUILD_TYPE STREQUAL "")
+    SET(CMAKE_BUILD_TYPE Release)
+ENDIF()
```

Comment:
This already has a pre-baked default of `Debug` and it should not be changed even if empty.  This setting should be controlled solely on the build command and not via a cmake file, thus these lines 25-27 should be removed.

Reply:
I removed them.